### ISSUE TITLE
Search should be more keyboard accessible

### ISSUE THREAD

**Issue by scorbo2** [scorbo2/snotes#6]
There's already a keyboard shortcut to bring up the search dialog, but there's no easy keyboard-only way of completing the search. You have to click on the "search" button, or tab tab tab and then space when the focus is on it, which is annoying. Maybe set it up such that hitting "enter" in the tag entry box executes the search.
